TSC-Charter.md

```diff
@@ -81,21 +81,30 @@ Subject to such policies as may be set by the Board, the TSC is
 responsible for all technical development within the Node.js Foundation,
 including:
 
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and any coding standards.
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Maintaining the list of additional Collaborators
```

Comment:
I know it was here before, but what exactly does the "additional" bit in "additional collaborators" mean?

Reply:
I guess Collaborators who are not on the TSC.